### ISSUE TITLE
Not working with vee-validate

### ISSUE THREAD

**Issue by cody-baker** [wuori/vue-credit-card-validation#13]
Here is what I have. When the input is rendered `formatCardNumber` is not working at all. My input is just a normal input. However, without the `ValidationProvider` the formatter works just fine. The same is happening with `formatCardCVC`. Interestingly enough, I haven't had any issues with `restrictNumeric` working with vee-validate though

```html
<ValidationProvider
    v-slot="{errors}"
    tag="div"
    rules="required"
>
    <b-input-group>
        <b-form-input
            v-model="cardNumber"
            v-cardformat:formatCardNumber
        />

       <template slot="prepend">
           <b-input-group-text>
                 <FontAwesomeIcon
                     v-if="cardBrand && paymentTypeIcons[cardBrand]"
                     size="lg"
                     :icon="paymentTypeIcons[cardBrand]"
                  />
            </b-input-group-text>
        </template>
    </b-input-group>
</ValidationProvider>
```

**Comment by cody-baker** [comment]
Realized that removing `v-model="cardNumber"` fixes the formatter. Not sure why this would only be the case when wrapped by `ValidationProvider` though

**Comment by wuori** [comment]
Sorry, @cody-baker I haven't any experience with vee-validate. If you come up with a solution please feel free to make a PR. Thanks!

**Comment by wuori** [comment]
@cody-baker I pushed a fix just now (v1.0.9) that fixes a duplicate function name. MAY help with your issue but doubtful. Closing this as it seems you found a solution and this is to do with another plugin. Thank you for your feedback!
